ui/src/app/shared/utils/url-search-params.utils.ts

```diff
@@ -11,23 +12,38 @@ export class URLSearchParamsUtils {
    *  that can be passed as a QueryParam. */
   public static encodeURLSearchParams(request: QueryJobsRequest): string {
     let urlSearchParams = new URLSearchParams();
-    urlSearchParams.set(queryFields.parentId, request.parentId);
-    urlSearchParams.set(queryFields.jobName, request.name);
+
     for (let s in request.statuses) {
       urlSearchParams.append('statuses', JobStatus[request.statuses[s]]);
     }
+    if (request.name) {
+      urlSearchParams.set('name', request.name);
+    }
     if (request.start) {
-      urlSearchParams.set(queryFields.start, request.start.toISOString());
+      urlSearchParams.set('start', request.start.toISOString());
     }
     if (request.end) {
-      urlSearchParams.set(queryFields.end, request.end.toISOString());
+      urlSearchParams.set('end', request.end.toISOString());
+    }
+
+    if (request.extensions) {
+      if (request.extensions.projectId) {
+        urlSearchParams.set('projectId', request.extensions.projectId);
+      }
+      if (request.extensions.userId) {
+        urlSearchParams.set('userId', request.extensions.userId);
+      }
+      if (request.extensions.submission) {
+        urlSearchParams.set('submission', request.extensions.submission.toISOString());
```

Comment:
Did you meant o include the submission-related stuff in this PR? Is that supported on the server yet?

Reply:
Yup, I did. I think the UI should support populating the request with query fields and query extension fields as soon as they are added to the model. Our dsub backend does not yet support submission, but our capaibilites endpoint also doesn't return it as a `query_extensions` so it won't be displayed (that actually wasn't true but I just changed it :+1:)